hbase_storage/src/main/java/com/splicemachine/si/impl/CoprocessorTxnStore.java

```diff
@@ -334,9 +337,22 @@ public long commitCount(){
      * **************************************************************************************************************
      */
         /*private helper methods*/
-    private TxnView decode(TxnMessage.Txn message) throws IOException{
+    private TxnView decode(long queryId, TxnMessage.Txn message) throws IOException{
         TxnMessage.TxnInfo info=message.getInfo();
```

Comment:
Is queryId begin timestamp of a txn?

Reply:
Yes, the txnId/begin timestamp. When we create a mocked committed transaction it is as if the transaction committed at the same point in time it started.